src/betterproto/streams/__init__.py

```diff
@@ -0,0 +1,82 @@
+from .. import Message, encode_varint, decode_varint
+from io import BufferedReader, BufferedWriter
+from typing import Iterable, Optional
+
+
+class CodedOutputStream:
+    stream: BufferedWriter
+
+    def __init__(self, stream: BufferedWriter) -> None:
+        self.stream = stream
+
+    def write_varint(self, value: int) -> None:
+        """Write a single varint value to the stream."""
+        self.stream.write(encode_varint(value))
+
+    def write_message(self, message: Message) -> None:
+        """Write a serialized message to the stream."""
+        self.stream.write(bytes(message))
+
+    def write_messages(self, messages: Iterable[Message]) -> None:
+        """Write multiple messages to the stream."""
+        for message in messages:
+            # Write the size of the message to delimit it and enable parsing
+            self.write_varint(len(bytes(message)))
```

Comment:
Is it better to implement `Message.write_delimited_to` by analogy with Java and use it here?

Reply:
Done